extensionblocker.js

```diff
@@ -0,0 +1,38 @@
+// Note: jQuery needed
+var result;
+var allowedExtensions = [];
+function checkExt(name,codesite,downloadsite) {
+var datavar;
+$.get(codesite,function(data,status){datavar=data;
```

Comment:
1. scratch-www does not use jQuery
2. add a space after "function"
3. remove the "status" argument (or if something will actually be passed in, rename it to `_`)

Reply:
Why doesn't it use jQuery? I think it would actually be helpful. I guess I have to use vanilla JS and make my own API library...